### DISCUSSION TITLE
好奇一下贵群内群主是否本人?请问问题被吊后被踢?

### DISCUSSION THREAD

**Discussion by xiaojiejieouo** [EstrellaXD/Auto_Bangumi#634] [General]
笑死

**Comment by xiaojiejieouo** [comment]
如果你真是管理员或者群主希望能教好群员礼仪, 请问问题被吊不会用可以不用, 我说不会说话可以不说.就被踢是吧, 博士,在你是管理or群主的前提下真是刷新了我对阁下的认知

**Comment by EstrellaXD** [comment]
谢谢提醒，已经提醒各位友善对待新来成员啦，不过还请您多反思一下为什么建群以来只有您和广告号被踢了。

**Reply by EstrellaXD** [reply]
顺便看了下聊天记录
<img width="267" alt="image" src="https://github.com/EstrellaXD/Auto_Bangumi/assets/33726646/b155b356-01e7-4ec9-8000-8c5f66f1dfd9">
进群第一件事情先伸手
<img width="262" alt="image" src="https://github.com/EstrellaXD/Auto_Bangumi/assets/33726646/b5fb160d-0129-48ff-a0ec-1134e0adbbf2">
「不会说话可以不说」
<img width="229" alt="image" src="https://github.com/EstrellaXD/Auto_Bangumi/assets/33726646/b23a13f4-aa4f-42b4-aa2a-624308eecf85">
不看文档又不懂礼貌确实活该
<img width="276" alt="image" src="https://github.com/EstrellaXD/Auto_Bangumi/assets/33726646/6c719cd8-236e-45f4-bd1f-0b3da57d9b0d">
您说您该不该被踢呢
